src/components/experiment-tracking/runs-list-card/runs-list-card.scss

```diff
@@ -17,7 +17,7 @@
 
   &--active {
     background: var(--color-run-list-hover);
-    box-shadow: -1px 0px 0px inset $color-accent-blue;
+    border-right: solid #00bcff;
```

Comment:
This color will be set in the `variables.scss` file, after the design team confirms it.